http/README.md

```diff
@@ -68,5 +68,84 @@ Table below shows the behaviour of HttpAction depending on provided `responseOpt
 | application/text | false     | JSON           | JSON | _error     | -        |
 | application/text | true      | JSON           | JSON | _error     | -        |
 
+### Node log
+`HttpAction` provides node log, every HTTP call is logged on `INFO` level, failing ones also on `ERROR`. 
+Every HTTP call which have ended with 200 status, but causing exception during processing in `HttpAction` 
+will be treated as succeeding, and the transition will be set to `_error`. Every error is logged.
+
+Node log is presented as `JSON`. 
+
+`JSON` structure of succeeding request on `INFO` level looks like the following (on `ERROR`
+level no logs will be produced):
+```json
+{
+  "request": {
+    "path": "/api/endpoint",
+    "requestHeaders": {
+      "Content-Type": "application/json"
+    }
+  },
+  "response": {
+    "httpVersion": "HTTP_1_1",
+    "statusCode": "200",
+    "statusMessage": "OK",
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "trailers": {
+    },
+    "httpMethod": "GET",
+    "requestPath": "http://localhost/api/endpoint"
+  },
+  "rawBody": "raw body received from service",
+  "result": {
+  }
+}
+```  
+`JSON` structure of succeeding request, but causing exception during processing on `INFO` level 
+looks like the following (on `ERROR` level log looks almost the same, every node except `rawBody` is logged,
+other entries remain the same):
+```json
+{
+  "request": {
+    "path": "/api/endpoint",
+    "requestHeaders": {
+      "Content-Type": "application/json"
+    }
+  },
+  "response": {
+    "httpVersion": "HTTP_1_1",
+    "statusCode": "200",
+    "statusMessage": "OK",
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "trailers": {
+    },
+    "httpMethod": "GET",
+    "requestPath": "http://localhost/api/endpoint"
+  },
+  "rawBody": "raw body received from service",
+  "error": {
+    "className": "io.reactivex.exceptions.CompositeException",
```

Comment:
Is it correct that we log `CompositeException` here?

> Represents an exception that is a composite of one or more other exceptions. A {@code CompositeException} does not modify the structure of any exception it wraps, but at print-time it iterates through the list of Throwables contained in the composite in order to print them all.

It is more like a list of exceptions... so maybe we should handle it as a list of exceptions (see `getExceptions()` method).

So maybe we should keep not one but more exceptions:
```
"errors" : [
  {
    "className": "DecodeException",
    "message": "This value....."
  },
  {
    "className": "SomeOtherException",
    "message": "This value....."
  }
]
```